src/app/googDevice/client/StreamClientScrcpy.ts

```diff
@@ -413,20 +491,24 @@ export class StreamClientScrcpy
         return this.deviceName;
     }
 
-    public setHandleKeyboardEvents(enabled: boolean): void {
+    public setHandleKeyboardEvents(enabled: boolean, currentWindow?: CurrentWindow): void {
+        this.keyHandlerEnabled = enabled;
+        currentWindow ??= CurrentWindow.pipWindow ?? CurrentWindow.main;
+
         if (enabled) {
-            KeyInputHandler.addEventListener(this);
+            KeyInputHandler.addEventListener(currentWindow, this);
         } else {
-            KeyInputHandler.removeEventListener(this);
+            KeyInputHandler.removeEventListener(currentWindow, this);
         }
     }
 
     public onKeyEvent(event: KeyCodeControlMessage): void {
         if (event.metaState & KeyEvent.META_META_ON) {
             if (event.keycode === KeyEvent.KEYCODE_V) {
                 const sendClipboard = async () => {
+                    const { currentWindow } = CurrentWindow.pipWindow ?? CurrentWindow.main;
```

Comment:
Maybe add a static func to do this logic?  I see you use this logic in multiple places: `CurrentWindow.pipWindow ?? CurrentWindow.main`

Reply:
Agreed, PR here: https://github.com/PetrasRec/ws-scrcpy/pull/18